formats/summary_test.go

```diff
@@ -1,199 +1,413 @@
 package formats
 
 import (
+	"sort"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
-func TestSummaryCount(t *testing.T) {
+func TestCuratedPackages(t *testing.T) {
 	testCases := []struct {
-		name     string
-		count    SummaryCount
-		expected int
+		name             string
+		curatedPackages  CuratedPackages
+		expectedApproved int
+		expectedBlocked  int
 	}{
-		{"Empty", SummaryCount{}, 0},
-		{"Single", SummaryCount{"High": 1}, 1},
-		{"Multiple", SummaryCount{"High": 1, "Medium": 2, "Low": 3}, 6},
+		{"Empty", CuratedPackages{}, 0, 0},
+		{"Approved", CuratedPackages{PackageCount: 1}, 1, 0},
+		{"Blocked", CuratedPackages{Blocked: []BlockedPackages{{Packages: map[string]int{"npm://test:1.0.0": 1}}}, PackageCount: 1}, 0, 1},
+		{
+			"Multiple",
+			CuratedPackages{
+				Blocked: []BlockedPackages{
+					{
+						Policy:    "Test",
+						Condition: "Test condition",
+						Packages:  map[string]int{"npm://test:1.0.0": 1},
+					},
+					{
+						Policy:    "Test2",
+						Condition: "Test condition 2",
+						Packages:  map[string]int{"npm://test2:1.0.0": 1, "npm://test3:1.0.0": 1},
+					},
+				},
+				PackageCount: 9,
+			},
+			6, 3,
+		},
 	}
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			assert.Equal(t, testCase.expected, testCase.count.GetTotal())
+			assert.Equal(t, testCase.expectedApproved, testCase.curatedPackages.GetApprovedCount())
+			assert.Equal(t, testCase.expectedBlocked, testCase.curatedPackages.GetBlockedCount())
 		})
 	}
 }
 
-func TestTwoLevelSummaryCount(t *testing.T) {
+func TestResultSummary(t *testing.T) {
+	testSummary := ResultSummary{
+		"High":    map[string]int{NoStatus: 1, "Status1": 2},
```

Comment:
Please be consistent with the quotes

Reply:
?

Reply:
Add quotes to "NoStatus" - not only to "Status1"